Dockerfile

```diff
@@ -6,7 +6,7 @@ VOLUME /config
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-RUN pip3 install --no-cache-dir colorlog cython
+RUN pip3 install --no-cache-dir colorlog cython mysqlclient psycopg2 uvloop
```

Comment:
All three requirements are optional (for DB and uvloop is a specific async library used on some platforms, like Docker). 

Lets just keep their install after the requirements.txt

The requirements I'm talking about are the requirements of `uvloop` for example, it requires `aiohttp` (any version), and we pin `aiohttp`